public/less/themes/arc-green.less

```diff
@@ -1294,8 +1294,34 @@ a.ui.labels .label:hover {
     }
 }
 
+.heatmap(@heat) {
+    @heatmap-cold: #2d303b;
+    @heatmap-hot: #87ab63;
```

Comment:
Maybe we can use the color for a:hiver (#a0cc75) which is a little brighter ? I feel like the heatmap is too dark now 😄

Reply:
And how #00FF48 looks? (i can't imagine)

Reply:
Good point 👍

Reply:
With `#a0cc75` from `a:hover` (I like it):
![Screenshot 2019-11-06 at 16 08 59](https://user-images.githubusercontent.com/136339/68310740-cbb37000-00b0-11ea-9e6f-5ff996388fed.png)

With `#00ff48` it looks out of place IMHO… too vibrant compared to the rest of the theme:
![Screenshot 2019-11-06 at 16 12 08](https://user-images.githubusercontent.com/136339/68310704-bcccbd80-00b0-11ea-8803-c86191c94ab9.png)